### DISCUSSION TITLE
Having issue deploying parse server app on Dokku

### DISCUSSION THREAD

**Discussion by armaneth95** [dokku/dokku#4773] [Q&A] [answered]
Hi
I just discovered dokku and give it a try. That's awesome to be able to deploy so easily.

Now am trying to deploy an app developed using Nuxt JS for the frontend, Parse Server for backend and MongoDB.
This work quite fine on Heroku when the dev team was working. I also could deploy it on docker but am failing to deploy it on Dokku.
The deployment successfully proceed but when I launch the URL Dokku provide for the app I get: 404 not found from NGINX
The steps I followed are those given here https://dokku.com/docs/deployment/application-deployment/

Any help or guidance is welcome.
Thanks in advance

**Comment by josegonzalez** [comment]
Can you show the output of the deploy? Additionally, is your app listening to the $PORT variable on all interfaces, and is it using a Dockerfile?

**Comment by armaneth95** [comment, marked as answer]
After git push everything run correctly and I get the application URL. My
main issue was that I thought that main URL will be enough to access the
app. Finally the developer am working with just forget to tell me which
path I had to add to that URL to access the app.

I almost feel stupid for opening this ticket. But it's good to know parse
server apps can be deploy on Dokku successfully.

Thanks for your help
Regards
--
Kenneth Hounsou

**Reply by josegonzalez** [reply]
No worries, these things happen from time to time. Happy you got things resolved :)
